tests/utils/managed_process.py

```
@@ -0,0 +1,242 @@
+# limitations under the License.
+
+import logging
+import os
+import shutil
+import socket
+import subprocess
+import time
+from dataclasses import dataclass, field
+from typing import Any, List, Optional
+
+import psutil
```

Pylint (3.3.7)

[error] Unable to import 'psutil'

(E0401)